.github/workflows/inspektor-gadget.yml

```diff
@@ -290,14 +290,26 @@ jobs:
         # - latest commit of btfhub
         # - hash of all bpf objects
         key: btfgen-${{ matrix.platform }}-${{ steps.get-btfhub-head.outputs.head }}-${{ hashFiles('pkg/**/*.o') }}
+    - name: Set ebpf-builder repository and determine image tag
+      if: startsWith(github.ref_name, 'v')
+      id: set-ebpf-builder-repo-and-determine-image-tag
+      uses: ./.github/actions/set-container-repo-and-determine-image-tag
+      with:
+        registry: ${{ env.REGISTRY }}
+        container-image: ${{ github.repository_owner }}/ebpf-builder
+    - name: Set ebpf-builder image name
+      if: startsWith(github.ref_name, 'v')
+      id: set-ebpf-builder-image-name
+      run: |
+        echo "name=${{ steps.set-ebpf-builder-repo-and-determine-image-tag.outputs.container-repo }}:${{ steps.set-ebpf-builder-repo-and-determine-image-tag.outputs.image-tag }}" >> $GITHUB_OUTPUT
```

Comment:
I'm wondering if we can reuse `build-helper-images` instead of creating a different step to calculate those tags. Probably it'll need to have build-helper-images as a dependency for this ig build job, but perhaps it's not too bad? 

Just take it a some idea I have, if you think it's not worth it I'm totally fine.

Reply:
> but perhaps it's not too bad?

I like the idea. It avoid the extra steps in each job. Also realized that we need the same for `ig img` so works well for that as well!